.yarnrc.yml

```diff
@@ -0,0 +1,4 @@
+nodeLinker: node-modules
```

Comment:
It would be nice to make it also compatible with anything else (like `nodeLinker: pnpm`).

In principle there's nothing stopping us from being neutral to the node_modules layout. I think there's just one place that hardcodes a node_modules path for resolution, and that can be solved.

Reply:
I'm interested in trying that too, but we should do it as a separate step after merging this, just so it's easier to identify the cause of any problems.

Reply:
Yes, it should be a separate step. I actually started to test with nodeLinker: pnpm and got into problems with the plone scripts chalk dependency. I think, even if we don't use it by default, working on pnpm compatibility will result in an overall cleaner stack.